crates/sui-config/tests/snapshots/snapshot_tests__populated_genesis_snapshot_matches-4.snap

```diff
@@ -101,14 +103,16 @@ validators:
       net_address: []
       next_epoch_stake: 1
       next_epoch_delegation: 0
+      next_epoch_gas_price: 1
 treasury_cap:
-  value: 100000000000001
+  value: 0
 storage_fund:
-  value: 100000000000000
+  value: 100000000000001
 parameters:
   min_validator_stake: 100000000000000
-  max_validator_candidate_count: 100
-  storage_gas_price: 1
+  max_validator_candidate_count: 100000000000000
+  storage_gas_price: 100
 delegation_reward:
-  value: 0
+  value: 1
+reference_gas_price: 0
```

Comment:
It seems that the values here are shifted up by one for some reason? It's probably caused by the addition of the new field `reference_gas_price`. How can I fix this? @mwtian

Reply:
Hmm I suspect this can be a real issue. Potentially there is an incompatibility between [serialization](https://github.com/MystenLabs/sui/blob/036462af58a6acf6ec88465bf386b1164d70dccd/crates/sui-config/src/genesis.rs#L463) and [deserialization](https://github.com/MystenLabs/sui/blob/036462af58a6acf6ec88465bf386b1164d70dccd/crates/sui-config/src/genesis.rs#L101) of the `SuiSystemObject` in the genesis blob. @bmwill may have more intuition here. I can help take a look tomorrow too.

Reply:
My best guess would be the arguments passed in to create the system object are passed in an incorrect order.

Reply:
Thanks! Ya there's likely a mistake somewhere. Will double check.